apps/central-scan/backend/src/cvrs/export.ts

```diff
@@ -25,16 +25,13 @@ import {
 
 function isHmpbPage(
   interpretation: PageInterpretation
-): interpretation is HmpbPageInterpretation {
-  return (
-    interpretation.type === 'InterpretedHmpbPage' ||
-    interpretation.type === 'UninterpretedHmpbPage'
-  );
+): interpretation is InterpretedHmpbPage {
+  return interpretation.type === 'InterpretedHmpbPage';
```

Comment:
how are these helpers used? Is this going to change whether or not these pages are exported if this now returns false instead of true?